wikiconv/conversation_reconstruction/dataflow_main.py

```diff
@@ -15,139 +15,230 @@
 A dataflow pipeline to reconstruct conversations on Wikipedia talk pages from
 ingested json files.
 
-Run with:
+Run on local test data with:
+
+```
+# Run locally:
+python dataflow_main.py \
+  --setup_file ./setup.py \
+  --input_state './testdata/empty_init_state' \
+  --input_revisions './testdata/edgecases_28_convs/revs*' \
+  --output_state ./tmp/ \
+  --output_conversations ./tmp/ \
+  --runner DirectRunner
+
+# Run on the cloud with:
+python dataflow_main.py \
+  --setup_file ./setup.py \
+  --input_state 'gs://YOUR_BUCKET/PATH_TO_INIT_STATE' \
+  --input_revisions 'gs://YOUR_BUCKET/PATH_TO_REVS*' \
+  --output_state 'gs://YOUR_BUCKET/PATH_TO_OUTPUT_STATE_TO' \
+  --output_conversations 'gs://YOUR_BUCKET/PATH_TO_OUTPUT_CONVS_TO' \
+  --runner DirectRunner
+  --project $YOUR_GCLOUD_PROJECT \
+  --job_name $NAME_OF_YOUR_JOB
+  --num_workers $NUMBER_OF_WORKERS_SUCH_AS_80
+```
```

Comment:
We need to change the parameters in helper shell accordingly.

Reply:
Also I think num_workers is "suggested number of workers", if we want to constrain it to max_num_workers, we need to change the code. I personally found max_num_workers was more useful.

Reply:
Thanks, I'll put in --max_num_workers into the suggested run command too.